avaframe/com2AB/com2ABCfg.ini

```diff
@@ -24,23 +24,18 @@ distance = 10
 # dsMin meters after the beta point also have an angle bellow 10°
 dsMin = 30
 
+# Optionnal (if left blanc, the avalanche path is read from Inputs/LINES/*AB*.shp and
+# the splitPoint from Inputs/POINTS/*.shp)
+# provide the system path to the avalanche path (relative path where . is avaframe)
```

Comment:
Would also work with a full path, right?

Reply:
not sure about that, should I try, should it work with the full path?

Reply:
Full path works too